### PR TITLE
adding PHP linux packages info

### PR DESCRIPTION
**Pull request by brettmc**

### CODE REVIEW

File: content/en/docs/instrumentation/php/automatic.md [hunk 1]
```diff
@@ -79,7 +79,43 @@ function will record it, without affecting exception propagation.
...
+- [Remi repository](https://blog.remirepo.net/pages/PECL-extensions-RPM-status) -
+  RPM
+- [Alpine linux](https://pkgs.alpinelinux.org/packages?name=*pecl-opentelemetry) -
+  APK (currently in the
+  [_testing_ branch](https://wiki.alpinelinux.org/wiki/Repositories#Testing))
+
+{{< tabpane text=true >}} {{% tab "RPM" %}}
+
+```sh
+#this example is from centos 7
```
Comment (svrnm) [inline content/en/docs/instrumentation/php/automatic.md:98]:
```suggestion
#this example is for centos 7
```

File: content/en/docs/instrumentation/php/automatic.md [hunk 2]
```diff
@@ -79,7 +79,43 @@ function will record it, without affecting exception propagation.
...
+yum install -y php php-pecl-opentelemetry
+
+php --ri opentelemetry
+```
+
+{{% /tab %}} {{% tab "APK" %}}
+
+```sh
+echo "@testing https://dl-cdn.alpinelinux.org/alpine/edge/testing" >> /etc/apk/repositories
+apk add php php81-pecl-opentelemetry@testing
```
Comment (svrnm) [inline content/en/docs/instrumentation/php/automatic.md:112]: is there a reason why this is for PHP 8.1 and the RPM for PHP 8.2? Do both exist? Might it be worthwhile to add a comment that people should make sure here to pick the right version for the PHP they use?

File: content/en/docs/instrumentation/php/automatic.md [hunk 3]
```diff
@@ -79,7 +79,43 @@ function will record it, without affecting exception propagation.
...
+(docker specific). There are also packaged versions of the extension available
+for some Linux package managers.
+
+### Linux packages
+
+RPM and APK packages are provided by the following:
+
+- [Remi repository](https://blog.remirepo.net/pages/PECL-extensions-RPM-status) -
+  RPM
+- [Alpine linux](https://pkgs.alpinelinux.org/packages?name=*pecl-opentelemetry) -
```
Comment (svrnm) [inline content/en/docs/instrumentation/php/automatic.md:91]:
```suggestion
- [Alpine Linux](https://pkgs.alpinelinux.org/packages?name=*pecl-opentelemetry) -
```

**Review by svrnm — CHANGES_REQUESTED** [review]

File: content/en/docs/instrumentation/php/automatic.md (hunk 2 above)
Comment (brettmc) [reply content/en/docs/instrumentation/php/automatic.md:112]: They do both exist, I'll add a note about it.

Comment (svrnm) [reply content/en/docs/instrumentation/php/automatic.md:112]: thanks

**Review by svrnm — APPROVED** [review]

**Comment by svrnm** [comment]
TY!

**Pull request merged**